File: API/Entities/Job.cs
using System;
using System.Collections.Generic;
using System.ComponentModel.DataAnnotations;
using System.ComponentModel.DataAnnotations.Schema;
using System.Linq;
using System.Threading.Tasks;

namespace API.Entities
{
    [Table("Jobs")]
    public class Job
    {
         public int Id{get;set;}
         public string Title{get;set;}
        public int ConfirmedOrgId {get;set;}
        public Organization Organization{get;set;}
        public AppUser JobPoster { get; set; }
        public string Description{get;set;}
        public int Salary{get;set;}
        public string City{set;get;}
        public string ProvinceOrState{get;set;}
        public string Country{get;set;}
        public string Genres{get; set;}
        public string JobType {get;set;}
        public string SkillsRequired {get; set;}
        public string ApplicationUrl {get;set;}
        public DateTime DateCreated{get;set;} = DateTime.Now;
        public DateTime Deadline{get;set;} = DateTime.Now;
        public DateTime LastUpdated{get;set;} = DateTime.Now;
        public ICollection<JobSave> SavedByUsers{get;set;}  
    }
}

File: client/src/app/_models/job.ts
import { Photo } from "./photo"
import { User } from "./user"

export interface Job {
    id: number
    title: string
    //Ogranization which posted this job
    //dependant on cap72
    confirmedOrgId: number
    jobPosterId: number
    jobPosterName: string
    logoUrl: string
    description: string
    salary: number
    city: string
    provinceOrState: string
    country: string
    genres: string
    jobType: string
    skillsRequired: string
    applicationUrl: string
    dateCreated: Date
    deadline: Date
    lastUpdated: Date




  }